flink-connector/flink/build.gradle.kts

```diff
@@ -29,7 +29,7 @@ repositories {
 var paimonVersion: String = libs.versions.paimon.get()
 val flinkVersion: String = libs.versions.flink.get()
 val flinkMajorVersion: String = flinkVersion.substringBeforeLast(".")
-
+val flinkJdbcVersion: String = "3.2.0-1.18"
```

Comment:
could you move the flink jdbc version in `gradle/libs.versions.toml`?

Reply:
fixed